src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/api/DeltakerApi.kt

```diff
@@ -201,8 +201,9 @@ fun Routing.registerDeltakerApi(
             val enheter = navEnhetService.hentEnheterForHistorikk(historikk)
 
             val arrangornavn = deltaker.deltakerliste.arrangor.getArrangorNavn()
+            val tiltakstype = deltaker.deltakerliste.tiltak.arenaKode
 
-            val json = objectMapper.writePolymorphicListAsString(historikk.toResponse(ansatte, arrangornavn, enheter))
+            val json = objectMapper.writePolymorphicListAsString(historikk.toResponse(ansatte, arrangornavn, enheter, tiltakstype))
```

Comment:
Jeg synes det virker litt klønete å sende dette inn i hvert vedtak, når du mest sannsynligvis kan utlede dette i frontend fra før, siden du har en deltaker i kontekst og kan sende inn deltakerlisten igjennom props der.  Men hvis det er tungvint så er dette ok.

Skulle ønske vi kunne brukt den andre typen tiltakskode som er litt mer moderne og lettere å forstå, men det krever sikkert større endringer i frontend.

Reply:
hmm, jeg tok den vi får for andre tiltaksstyper, for da får vi arenakoden, men vil vi endre den?